infrastructure/ai/providers/openai_provider.py

```
@@ -78,7 +78,9 @@ async def generate(
                 except openai.NotFoundError as e:
                     logger.info(f"Responses API unsupported for {base_url}, falling back to chat completions: {str(e)}")
                     self.__class__._fallback_to_chat_cache.add(base_url)
-                    # 往下执行降级到 Chat Completions
+                except self.EmptyResponseError as e:
+                    logger.warning(f"Responses API returned empty content for {base_url}, falling back to chat completions: {str(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)